### PR TITLE
colord: 1.4.2 -> 1.4.4

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Notable changes

* multiple outputs
* enable all docs
* installed tests

###### News Items
https://github.com/hughsie/colord/blob/1.4.3/NEWS
https://github.com/hughsie/colord/blob/1.4.4/NEWS

###### Possible improvements
1. Normal tests should probably be enabled
2. NixOS test for the installed tests
3. Possibly have `colord` user for the daemon instead of [running as root](https://github.com/hughsie/colord/blob/1.4.4/meson_options.txt#L14)
   This would probably be a good follow up.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/colord/default.nix [hunk 1]
```diff
@@ -1,32 +1,83 @@
...
-    "-Denable-sane=true"
-    "-Denable-vala=true"
     "--localstatedir=/var"
-    "-Denable-bash-completion=true"
-    # TODO: man page cannot be build with docbook2x
-    "-Denable-man=false"
-    "-Denable-docs=false"
+    "-Dinstalled_tests=true"
+    "-Dlibcolordcompat=true"
+    "-Dsane=true"
```
Comment (worldofpeace) [inline pkgs/tools/misc/colord/default.nix]: Can't remember if there were problems with sane here.

File: pkgs/tools/misc/colord/default.nix [hunk 2]
```diff
@@ -1,32 +1,83 @@
...
+    "-Dvapi=true"
   ];
 
-  nativeBuildInputs = [ meson pkgconfig vala_0_40 ninja gettext libxml2 gobject-introspection makeWrapper ];
+  nativeBuildInputs = [
+    docbook_xsl
```
Comment (jtojnar) [inline pkgs/tools/misc/colord/default.nix]: Hmm, I should have probably ported the api docs to Docbook 5 too.

**Comment by jtojnar** [comment]
The installed test for NixOS should be easy to do: https://github.com/NixOS/nixpkgs/issues/34987

**Comment by worldofpeace** [comment]
Had to add a [patch](https://github.com/NixOS/nixpkgs/blob/5b9b4257529b6323ba8c1fccb266e1c8adebc45a/pkgs/tools/misc/colord/installed-tests-path.patch) for the tests.

I've ran them, think it was a success.

**Comment by worldofpeace** [comment]
Just saw this pr https://github.com/NixOS/nixpkgs/pull/56504
Interesting, don't like how it tries to access a network resource.

**Review by jtojnar — APPROVED** [review]

**Pull request merged**